docs/faq.md

```diff
@@ -24,39 +24,10 @@ View Models are a Redux-specific extension to these mechanisms. View models are
 **A**: Implement a [custom Read Model](read-side.md#custom-read-models). Custom Read Models allow you to use custom logic to communicate with a Read Model store.
 
```

Comment:
Please change the following lines above:

Line 4:
`description: This article provides answers to frequently asked questions about the reSolve framework.`
Please delete "provides" and "to":
`description: This article answers frequently asked questions about the reSolve framework.`

Line 23:
`**Q**: How to implement a Read Model with direct access to the underlying store?<br/>`
Can we change this to the following?
`**Q**: How do I implement a Read Model with direct access to the underlying store?<br/>`

Reply:
Done